tests/feature_test.py

```diff
@@ -3,34 +3,64 @@
+restapi     disabled  enabled                                                                                        none
+sflow       disabled  enabled                                                                                        none
+snmp        enabled   enabled                                                                                        none
+swss        enabled   enabled                                                                                        none
+syncd       enabled   enabled                                                                                        none
+teamd       enabled   enabled                                                                                        none
+telemetry   enabled   enabled                                                                                        none
+"""
+
+show_feature_config_output="""\
+Feature     State     AutoRestart    Owner    no-fallback
+----------  --------  -------------  -------  -------------
+bgp         enabled   enabled        local    false
+database    enabled   disabled       local    false
+dhcp_relay  enabled   enabled        kube     false
+lldp        enabled   enabled        kube     false
+nat         enabled   enabled        local    false
+pmon        enabled   enabled        kube     false
+radv        enabled   enabled        kube     false
+restapi     disabled  enabled        local    false
+sflow       disabled  enabled        local    false
+snmp        enabled   enabled        kube     false
+swss        enabled   enabled        local    false
+syncd       enabled   enabled        local    false
+teamd       enabled   enabled        local    false
+telemetry   enabled   enabled        kube     false
 """
 
 show_feature_bgp_status_output="""\
-Feature    State    AutoRestart
----------  -------  -------------
-bgp        enabled  enabled
+Feature    State    AutoRestart    SystemState    UpdateTime    ContainerId    ContainerVersion    CurrentOwner    RemoteState
+---------  -------  -------------  -------------  ------------  -------------  ------------------  --------------  -------------
+bgp        enabled  enabled                                                                                        none
 """
 
 show_feature_bgp_disabled_status_output="""\
-Feature    State     AutoRestart
----------  --------  -------------
-bgp        disabled  enabled
+Feature    State     AutoRestart    SystemState    UpdateTime    ContainerId    ContainerVersion    CurrentOwner    RemoteState
+---------  --------  -------------  -------------  ------------  -------------  ------------------  --------------  -------------
+bgp        disabled  enabled                                                                                        none
+"""
+show_feature_snmp_config_owner_output="""\
+Feature    State    AutoRestart    Owner    no-fallback
```

Comment:
should change to Fallback, why no-fallback?

Reply:
yep

Reply:
done.
```
admin@sonic:~$ show feature config snmp
Feature    State    AutoRestart    Owner    fallback
---------  -------  -------------  -------  ----------
snmp       enabled  enabled        kube     true
admin@:sonic~$ 
```